web/src/containers/students/Student.js

```diff
@@ -99,6 +128,77 @@ function Student(props) {
     setOpen(false);
   };
 
+  /* -------------------------------------------------------------------------- */
+
+  const pending = (
+    <Box>
+      <Typography variant="h3">Request waiting for approval</Typography>
+    </Box>
+  );
+
+  const funding = (
+    <Box
+      className="left-tok"
+      width="fit-content"
+      marginLeft="auto"
+      marginRight="auto"
+    >
+      <Box pt="10%" marginBottom="10%">
```

Comment:
mb is equivalent to marginBottom. but you can use py to give padding in the y axis and remove marginBottom. you can also use values from the theme like so: py={2} (this can be changed in multiple lines)